doc/user/content/quickstarts/streaming.md

```diff
@@ -0,0 +1,202 @@
+
+### Create the sources
+
+Materialize provides public Kafka topics and a Confluent Schema Registry for its users. The Kafka topics contain data from a fictional eCommerce and receive updates every second. You will use the data in them to build the analytics for the dashboard.
+
+1. In your `psql` terminal, create a new schema:
+
+    ```sql
+    CREATE SCHEMA shop;
+    ```
+
+1. Create the connection to the Confluent Schema Registry:
+    ```sql
+    CREATE SECRET IF NOT EXISTS shop.csr_username AS '<TBD>';
+    CREATE SECRET IF NOT EXISTS shop.csr_password AS '<TBD>';
+
+    CREATE CONNECTION shop.csr_basic_http
+    FOR CONFLUENT SCHEMA REGISTRY
+    URL '<TBD>',
+    USERNAME = SECRET csr_username,
+    PASSWORD = SECRET csr_password;
+    ```
+
+1. Create the connection to the Kafka broker:
+
+    ```sql
+    CREATE SECRET shop.kafka_password AS '<TBD>';
+
+    CREATE CONNECTION shop.kafka_connection TO KAFKA (
+        BROKER 'TBD',
+        SASL MECHANISMS = 'SCRAM-SHA-256',
+        SASL USERNAME = 'TBD',
+        SASL PASSWORD = SECRET shop.kafka_password
+    );
+    ```
+
+1. Create the sources, one per Kafka topic:
+    <!-- ```sql
+    CREATE TABLE IF NOT EXISTS shop.users (
+        id INT,
+        email TEXT,
+        is_vip BOOLEAN,
+        created_at TIMESTAMP,
+        updated_at TIMESTAMP
+    );
+
+    CREATE TABLE IF NOT EXISTS shop.items (
+        id INT,
+        name TEXT,
+        category TEXT,
+        price INT,
+        inventory INT,
+        inventory_updated_at TIMESTAMP,
+        created_at TIMESTAMP,
+        updated_at TIMESTAMP
+    );
+
+    CREATE TABLE IF NOT EXISTS shop.purchases (
+        id INT,
+        user_id BIGINT,
+        item_id BIGINT,
+        status INT,
+        quantity INT,
+        purchase_price INT,
+        deleted BOOLEAN,
+        created_at TIMESTAMP,
+        updated_at TIMESTAMP
+    );
+
+    INSERT INTO shop.users VALUES (1, 'random@email.com', true, current_timestamp(), current_timestamp());
+    INSERT INTO shop.items VALUES (1, 'Random Random', 'Category', 230, 3, current_timestamp(), current_timestamp(), current_timestamp());
+    INSERT INTO shop.purchases VALUES (1, 1, 1, 1, 3, 10, false, current_timestamp(), current_timestamp());
+    ``` -->
+
+    ```sql
+    CREATE SOURCE purchases
+    FROM KAFKA CONNECTION shop.kafka_connection (TOPIC 'purchases')
+    FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY CONNECTION shop.csr_basic_http
+    WITH (SIZE = '3xsmall');
+
+    CREATE SOURCE items
+    FROM KAFKA CONNECTION shop.kafka_connection (TOPIC 'items')
+    FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY CONNECTION shop.csr_basic_http
+    WITH (SIZE = '3xsmall');
+
+    CREATE SOURCE users
+    FROM KAFKA CONNECTION shop.kafka_connection (TOPIC 'users')
+    FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY CONNECTION shop.csr_basic_http
+    WITH (SIZE = '3xsmall');
+    ```
+
+    <!-- Each topic contains different data:
```

Comment:
Thinking about it some more, it's probably worth it to include a section that is common to all guides about the data, creating a cluster for the quickstarts and so on, since all three (+ the get started) will use the exact same source.